curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b05367b59300bcb5f18ef.md

```diff
@@ -0,0 +1,42 @@
+---
+id: 657b05367b59300bcb5f18ef
+title: Task 1
+challengeType: 22
+dashedName: task-1
+---
+
+# --description--
+
+In English, you can use the verb `to be` to ask questions about feelings, states, or conditions. 
+When you want to ask such questions, you just swap the positions of the subject (like `you` or `he`) and the verb `to be` (like `are` or `is`).
+
+For example, to turn this sentence `You are a developer.` into a question, you switch `you` and `are`: 
+
+`Are you a developer?`
+
+Notice how the order changes to ask a question.
+
+Here are some more examples of turning statements into questions using `to be`. 
+
+`He is here.` -> `Is he here?`
+
+`They are ready.` -> `Are they ready?`
+
+# --instructions--
+
+Listen to the audio to complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence-- 
+
+`Sarah: Hi, Tom! _ you happy with the workplace?`
```

Comment:
I was having issues filling in the blank when running this locally

<img width="849" alt="Screenshot 2023-12-16 at 9 17 53 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/3c03b06e-4a5b-43dc-8f7e-5a1e5c236c23">